docs/source/api/v4/roles.rst

```diff
@@ -327,3 +327,5 @@ Response Structure
 		"text": "role was deleted.",
 		"level": "success"
 	}]}
+
+.. [#permissions] ``permissions`` cannot include permissions that are not included in the permissions of the requesting user.
```

Comment:
This footnote is never referenced, it should be applied to applicable request and/or response properties